tests/test_JPEG_LS_transfer_syntax.py

```diff
@@ -120,7 +120,7 @@ def test_read_mr_with_gdcm(self):
         b = self.mr_small.pixel_array
         assert a.mean() == b.mean(), \
             "using GDCM Decoded pixel data is not " \
-            "all {0} (mean == {1})".format(b.mean(), a.mean())
+            "all {} (mean == {})".format(b.mean(), a.mean())
```

Comment:
why not convert to f-string? (there have been a few other more complicated instances, but this one looks quite simple)

Reply:
Yes, strangely the auto-fix tool converted many `.format` to f-string, but not all of them.  I updated these ones, rewrote the lines so not so complicated.